src/util/cmdlineargs.cpp

```diff
@@ -73,6 +74,23 @@ bool CmdlineArgs::Parse(int &argc, char **argv) {
 warnings and errors to the console unless this is set properly.\n", stdout);
             }
             i++;
+        } else if (argv[i] == QString("--logFlushLevel") && i+1 < argc) {
```

Comment:
...then we should use the term "logFlushLevel" consistently and not only "flushLevel" like in "kFlushLevelDefault" and some other places.